rssparser/src/commonMain/kotlin/com/prof18/rssparser/RssParser.kt

```diff
@@ -0,0 +1,48 @@
+package com.prof18.rssparser
+
+import com.prof18.rssparser.internal.XmlFetcher
+import com.prof18.rssparser.internal.XmlParser
+import com.prof18.rssparser.model.RssChannel
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.SupervisorJob
+import kotlinx.coroutines.withContext
+import kotlin.coroutines.CoroutineContext
+
+class RssParser internal constructor(
+    private val xmlFetcher: XmlFetcher,
+    private val xmlParser: XmlParser,
+) {
+
+    private val coroutineContext: CoroutineContext =
+        SupervisorJob() + Dispatchers.Default
+
+    internal interface Builder {
+        /**
+         * Creates a [RssParser] object
+         */
```

Comment:
I don't think this API should be documented like this. 😄 (The documentation is redundant.)